cabal.project

```diff
@@ -39,3 +38,23 @@ program-options
 package bitvec
   -- Workaround for windows cross-compilation
   flags: -simd
+
+if impl(ghc >=9.12)
+  allow-newer:
+    -- Stuck on `cabal-3.14` issues and maintainer difficulties
+    -- https://github.com/haskell/aeson/issues/1124
+    -- Stuck on `cabal-3.14` and unresponsive maintainers
+    -- https://github.com/well-typed/cborg/pull/339
+    -- Fix blocked by aeson
+    -- https://github.com/haskellari/microstache/issues/43
+    -- Difficult maintainer
```

Comment:
:smiling_face_with_tear:

Reply:
Hoping this can still be merged with these `allow-newer` stanzas. No idea when any of these will be fixed.